ext/snowcrash/src/MSON.h

```diff
@@ -197,7 +229,7 @@ namespace mson
     struct Element;
 
     /** Collection of elements */
-    typedef std::vector<Element> Elements;
+    typedef boost::container::vector<Element> Elements;
```

Comment:
I changed this to `boost::container::vector`, just because it allows usage before a contained type is defined. In the past, `Element` was forced to hold a `std::vector<std::unique_ptr<Element>>`, now we hold `vector<Element>` directly.